test/blueprints/Do.Test.Blueprints.Service.Test/TestServiceNfrExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Newtonsoft.Json;
+
+namespace Do.Test;
+
+public static class TestServiceNfrExtensions
+{
+    public async static Task<dynamic?> DeserializeContentToDynamic(this HttpContent content)
```

Comment:
this is extension to `HttpContent` so no need to have content in the name, and with the `object` return let's rename it to `DeserializeObject`

Reply:
or even simpler `Deserialize`